apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/carin/C4BBClaimProfessionalAndNonClinicianCareTeamRole.java

```diff
@@ -10,7 +10,8 @@
   SUPERVISOR,
   PERFORMING,
   PURCHASED_SERVICE,
-  REFERRING;
+  REFERRING,
+  OTHER;
```

Comment:
In V1 it is other, should it be other operating in v2? http://hl7.org/fhir/us/carin-bb/STU1/CodeSystem-C4BBClaimCareTeamRole.html